regen-network/regen-web
# feat: APP-189 post secret link

## Description

https://regennetwork.atlassian.net/browse/APP-189?atlOrigin=eyJpIjoiYWRjMDY0MWRmNWQwNGUwYmEzZmRkOThjZTVhNzdiZmYiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2401--regen-marketplace.netlify.app/post/regen:13toVg8bPSA85BdNcn7GibTNUoFD25q9krJRV7sWr2qPhyySgHMDv1A.rdf?token=7bbd0a3d7fde9293d9d1dcdb5b91284c
1. Create a post which private or has private files or locations (or use an existing one for a project you're an admin of) and go the post page while being logged in, you should have an option to share a secret link from the top right button "...".
2. Log out and use the copied secret link to access the post page, you should see all post info
3. Create a public post (or use an existing one for a project you're an admin of) and go to the post page while being logged in, the secret link option shouldn't be available.
4. Share one of your posts with some private data from the project data stream section

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
